postfix/README.md

```diff
@@ -78,6 +78,42 @@ List of users who are authorized to view the queue.
 
 [Restart the Agent][5] to start sending Postfix metrics to Datadog.
 
+#### Log Collection
+
+**Available for Agent >6.0**
+
+Postfix logs activity to the syslog daemon that then gets sent to logfiles.
+
+By default, the naming convention and log files can by configured in your syslog file:
+
+```
+/etc/syslog.conf:
+    mail.err                                    /dev/console
+    mail.debug                                  /var/log/maillog
+```
+
+
+* Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file with:
```

Comment:
```suggestion
* Collecting logs is disabled by default in the Datadog Agent. Enable it in your `datadog.yaml` file:
```

Reply:
This is present in a few other README's as well, as I took it from there. But I can change it to this :)

Reply:
The double sentences one is better since it's more "business". Simple sentence for simple concepts.